packages/eventual-send/src/local.js

```diff
@@ -0,0 +1,39 @@
+const q = JSON.stringify;
+
+const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
+
+export const localApplyFunction = (t, args) => {
+  if (!(t instanceof Function)) {
+    const ftype = ntypeof(t);
```

Comment:
`instanceof Function` is a terrible test. It is not a reliable indication of anything except that the object inherits from that `Function.prototype`. Here we care about whether it is callable, which is best approximated by

```suggestion
  const ftype = ntypeof(t);
  if (ftype !== 'function') {
```

Unfortunately, both tests will silently accept classes, which have a `[[Contruct]]` behavior but no `[[Call]]` behavior. The other choice is to let the call just fail if `t` is not callable. Was this put here because those builtin diagnostics are bad?

Reply:
It was an attempt to make errors more understandable, but not thoroughly tested since we don't yet rely on `applyFunction` in many (if any) places.

I'll adopt the `ftype !== 'function'` test.